src/main/java/model/Car.java

```diff
@@ -0,0 +1,52 @@
+package model;
+
+import java.util.Objects;
+
+public class Car {
+    private int position;
+    private final String name;
+    private static final int MAX_NAME_LENGTH = 5;
+    public static final int CAR_MOVE_THRESHOLD = 4;
+
```

Comment:
```suggestion

public class Car {
    private static final int MAX_NAME_LENGTH = 5;
    public static final int CAR_MOVE_THRESHOLD = 4;

    private int position;
    private final String name;

```

일반적으로 상수가 필드보다 위에 위치합니다.